dockerfile_scripts/scrape_libs.sh

```diff
@@ -1,12 +1,20 @@
 #!/bin/bash
 
+WHOAMI=$(whoami)
 # Only scrape host libs if AWS/NCCL/OFI was built for this image
 if [ -d /container/aws/lib ]
 then
    # See if we mounted in host libs in the expected location
-   host_dir="/host"
+   host_dir="/det_host"
    if [ -d "$host_dir" ]; then
-       libfabric=`find /host -name libfabric.so 2>/dev/null`
+       # to set these paths based on where we find the libs that
+       # libfabric is dependent upon.
+       export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$host_dir/usr/lib64
+       export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$host_dir/usr/local/lib64
+   fi # end if /det_host exists
+   host_dir="/det_libfabric"
+   if [ -d "$host_dir" ]; then
+       libfabric=`find $host_dir -name libfabric.so 2>/dev/null`
```

Comment:
Seems odd to search for it, if the intent of /det_libfabric is to provide libfabric.so

If /det_libfabric is present, should it instead error out with a message if libfabric.so is not found?   That would be a more usable UX.   They have expressed their intent by mounting the directory, so it seems if we ignore the failure it is just going to fail in an unexpected way later, or just not provide the performance that they are expecting.

Reply:
We are going to address this in a new PR.

Reply:
I've added an error message to explain that `libfabric.so` isn't found within the directory.